packages/manager/src/managers/telemetry/TelemetryManager.ts

```diff
@@ -268,4 +277,44 @@ export class TelemetryManager extends BaseManager {
 
 		return readPrismicrc(root).telemetry !== false;
 	}
+
+	private async initExperiment(): Promise<void> {
+		try {
+			const repositoryName = await this.project.getRepositoryName();
+
+			this._experiment = Experiment.initialize(API_TOKENS.AmplitudeKey, {
+				pollOnStart: false,
+				fetchOnStart: false,
+			});
+
+			await this._experiment
+				.start({
+					user_properties: {
+						Repository: repositoryName,
+					},
+				})
+				.catch((error) => {
+					console.error("Error starting experiment", error);
```

Comment:
why do you log this error and not the one below in prod?

Reply:
Good one! I think I just don't need a catch, better to just use the try catch for both, thanks